cloudflare/resource_cloudflare_access_group_test.go

```diff
@@ -0,0 +1,359 @@
+package cloudflare
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+var (
+	accountID   = "01a7362d577a6c3019a474fd6f485823"
```

Comment:
As this is creating real resources, this needs to be an account ID that is accessible and real. We should use an environment variable for configuration.

```suggestion
	accountID   = os.Getenv("CLOUDFLARE_ACCOUNT_ID")
```